### DISCUSSION TITLE
Next generations

### DISCUSSION THREAD

**Discussion by introvert** [prisma/prisma#24725] [Feedback]
### Bug description

Are you sure this a "next generation" ORM? It cannot be even set up with the basic postgresql providers after hours of buggling with the configuration.

Not even "db:push" can be properly completed in a normal dev environment, which makes us question whether we should continue bothering with the "next generation" project.

Are you guys sure that you know what meaning words have and the dissonance between the contemporary marketing of tooling vs reality of things?

### Expected behavior

Exactly as described.

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by apolanc** [comment]
Hi, this is not a very nice way to introduce yourself @introvert and appreciate what open source teams and projects offer the community and the effort put behind it.

Now, focusing on the actual problem: Could you explain the problem a little better and share some of the necessary information so that we can verify it and support you (environment and configuration, your schema file, etc.)? That would be greatly appreciated.

**Comment by dev-a-loper** [comment]
the way you define "next generation" is not  straight forward to me
 next generation doesn't mean no bugs and no problems it essentially means : a new approach which prisma indeed was a new approach ( altough its getting older these days specially in presence of drizzle )

**Comment by Zain-ul-din** [comment]
It seems like he is a hater coming from a PHP background, thinking everything is going to work magically.
